### PR TITLE
Style guide changes - DOC-6450

### PR DESCRIPTION
**Pull request by zuluecho9**
Docs team - Changes to article templates docs section for github migration.

### CODE REVIEW

File: src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/agent-api-guide-template.mdx [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+title: Agent API guide template
+tags:
+  - New Relic only
+  - Tech writer style guide
+  - Article templates
+redirects:
+  - /docs/new-relic-only/advanced-style-guide/article-templates/agent-api-guide-template
+---
+
+Introduction: The first section in such a doc is the introduction, which will introduce the agent and the API in general terms. Talk about when a user would want the API, and alternatives to using the API (for example, instrumentation via XML file). Mention that the API is often unnecessary if your framework has “out of the box” support. Link to the root of your API reference, whether that is on-site or off-site. Warn the user that they need to be on the most recent agent version.
```
Comment (urbiz-nr) [inline src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/agent-api-guide-template.mdx:11]: A lot of "introduction" in the first sentence.

File: src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/agent-api-guide-template.mdx [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+
+## Instrument missing sections of your code with transactions [#creating-transactions]
+
+Example text: To instrument your app, New Relic separates each path through your code into its own transaction. New Relic times (or “instruments”) the parent method in these transactions to measure your app’s overall performance, and collects transaction traces from long-running transactions for additional detail. For more information about transactions, see [transaction](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#transaction) and [transaction trace](/docs/apm/transactions/transaction-traces/introduction-transaction-traces).
```
Comment (urbiz-nr) [inline src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/agent-api-guide-template.mdx:15]: I'd remove the "New Relic" references, use "we" instead.

File: src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/agent-api-guide-template.mdx [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+Use these methods when New Relic is not instrumenting a particular part of your code at all:
```
Comment (urbiz-nr) [inline src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/agent-api-guide-template.mdx:17]: Are these methods up to date? I don't think this is markdown.

File: src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/agent-api-guide-template.mdx [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+      <td>
+        Mark the error as expected. See \[link to config doc or tutorial doc for expected_classes equivalent].
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Send custom event and metric data from your app [#custom-data]
+
+Example text: New Relic APM includes a number of ways to record arbitrary custom data. For an explanation of New Relic data types, see [Data collection](/docs/data-analysis/metrics/analyze-your-metrics/data-collection-metric-timeslice-event-data).
```
Comment (urbiz-nr) [inline src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/agent-api-guide-template.mdx:311]: Remove "New Relic" before APM.

**Comment by jpvajda** [comment]
is this still a valid PR?

**Comment by urbiz-nr** [comment]
I'd say so, let's see what Zach has to say.

**Comment by jpvajda** [comment]
@zuluecho9 can you confirm, thanks!

**Comment by bradleycamacho** [comment]
@zuluecho9 Is there a blocker on this?

**Comment by mmfred** [comment]
@zuluecho9 - please go ahead and merge this PR. Thanks!

**Comment by austin-schaefer** [comment]
Pretty sure the directory/nav on this will need changing— @paperclypse will know more, but I think the `new-relic-only` directory is going away.

**Pull request merged**